### PR TITLE
Added whiteflag test setup

### PR DESCRIPTION
**Pull request by alexsporn**
- Added the necessary helper functions and setup to create and visualize whiteflag confirmation
- Fixed a bug when calculating the address mutations during whiteflag confirmation

### CODE REVIEW

File: pkg/whiteflag/test/whiteflag_test.go [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+	for _, value := range balances {
+		sum += value
+	}
+	// Move remaining supply to 999..999
+	balances[string(hornet.NullHashBytes)] = consts.TotalSupply - sum
+
+	store := mapdb.NewMapDB()
+	setupTestEnvironment(t, store)
+
+	solidEntryPointsOnce.Do(func() {
```
Comment (muXxer) [inline pkg/whiteflag/test/whiteflag_test.go]: Why is it called `solidEntryPointsOnce` ?

File: pkg/whiteflag/test/whiteflag_test.go [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
+
+func TestWhiteFlagWithMultipleConflicting(t *testing.T) {
+
+	// Fill up the balances
+	balances := make(map[string]uint64)
+	balances[string(generateAddress(t, seed1, 0))] = 1000
+
+	milestones := setupCoordinatorAndIssueInitialMilestones(t, balances, 3)
+
+	// Issue some transactions
```
Comment (muXxer) [inline pkg/whiteflag/test/whiteflag_test.go:565]: Could you please add some comments here? It's hard to read the testcase :)
For example what would be the final balance you would expect on which seed, or something like that.

Comment (alexsporn) [reply pkg/whiteflag/test/whiteflag_test.go:565]: Do you mean as a comment of every bundle? At the end I check every seed

File: pkg/whiteflag/test/whiteflag_test.go (hunk 1 above)
Comment (alexsporn) [reply pkg/whiteflag/test/whiteflag_test.go]: forgot to rename it at the end, but actually it protects that we only itinialize the solid entry points once. We have some global state there that I cannot reset betweeen each test. I'd need to adapt the code, but we made it so to actually catch wrong usage. I'll rename the once

File: pkg/whiteflag/test/whiteflag_test.go (hunk 2 above)
Comment (alexsporn) [reply pkg/whiteflag/test/whiteflag_test.go:565]: OK, I see that without labels it looks difficult to read. My IDE was showing it to me like this:
Screenshot 2020-07-24 at 09 08 23

**Review by muXxer — APPROVED** [review]

**Pull request merged**